preemptive_requests.py

```diff
@@ -351,6 +396,52 @@ def update_valid_rate(self, value, label):
         valid_rate = get_valid_rate(value)
         label.setText(f'{valid_rate} Hz')
 
+    def update_jf_from_raw_trans(self, value: float, label: QtWidgets.QLabel) -> None:
+        """
+        Slot to recieve and use a new transmission readback.
+
+        This is combined with the incoming jf data to update
+        the effective transmission readback.
+        """
+        self.jf_trans_cache[label.channel] = value
+        self.update_jf_trans_for_label(label)
+
+    def update_jf_from_jf(self, value: float , label: QtWidgets.QLabel) -> None:
```

Comment:
Oh we don't have style checks/pre-commit running here, hmm?
```suggestion
    def update_jf_from_jf(self, value: float, label: QtWidgets.QLabel) -> None:
```

Reply:
I can turn these on now or I maybe in a separate PR I can reformat the whole repo

Reply:
Totally minor, just surprised me
Separate PR at any point is totally OK with me